js/NavMainComponent.js

```diff
@@ -87,16 +87,23 @@ NavMainComponent.prototype.init = function () {
     // Open quaternary nav on tertiary nav item click
     component.$tertiaryNavLinks.on('click', function (event) {
         var $self = $(this),
-            href = $self.attr('href');
+            target;
 
-        if (href.indexOf('#') !== -1) {
+        if ($self[0].hasAttribute('href')) {
```

Comment:
I think this `$self[0]` referencing is a bit messy, can we either drop `$(this)` and use `this` or use `$self` exclusively? So in this example instead of accessing the `HTMLElement` with the index, can't we just use `$self.attr('href')` and do away with the `hasAttribute`

Reply:
I mistakenly thought that `if ($self.attr('href')) {` would be problematic, but tests pass and behaviour seems fine